Firestore/Source/Util/FSTDispatchQueue.mm

```diff
@@ -145,20 +147,20 @@ - (void)markDone {
 
 #pragma mark - FSTDispatchQueue
 
-@interface FSTDispatchQueue ()
+@interface FSTDispatchQueue () {
```

Comment:
I can add an instance variable here, right?

Reply:
Yes, though customarily it's just done on the `@implementation`.

We use the empty category to make "private" declarations that must be in an `@interface`. Things like property declarations or a hidden designated initializer.

Reply:
Hmm, it appears that if I move the variable declaration inside `@implementation`, it becomes a global variable, not instance variable.

Reply:
The declaration must still be inside braces. See e.g.

https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/Source/Local/FSTLevelDBKey.mm#L471